src/pages/_app.tsx

```diff
@@ -57,14 +55,6 @@ Router.events.on(
   (url, { shallow }) => !shallow && NProgress.done()
 );
 
-const queryClient = new QueryClient({
-  defaultOptions: {
-    queries: {
-      refetchOnWindowFocus: false,
-    },
-  },
-});
-
```

Comment:
Glad to see that this was such a simple thing, once all the parts that depended on it were gone!